tox.ini

```diff
@@ -1,3 +1,6 @@
 [flake8]
 exclude = amgut/lib/locale_data/*,amgut/templates/*,amgut/static/*,amgut/db/*
-max-complexity = 10
\ No newline at end of file
+max-complexity = 10
+
+[pep257]
+ignore = D100,D102,D200,D203,D204,D205,D400
```

Comment:
I had my own `.pep257` file in the directory for the code I was adding; I set it to ignore some of the more pedantic errors. `pep257` is unfortunately written in a way that if there's a `tox.ini` file in the directory that it always looks for its settings there and no longer looks in the `.pep257` file.

Decision about whether you all want to use it or not is up to you. :)